src/Provider/UpdatedOrderProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Setono\SyliusOrderEditPlugin\Provider;
+
+use Sylius\Bundle\OrderBundle\Form\Type\OrderType;
+use Sylius\Component\Core\Model\OrderInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Webmozart\Assert\Assert;
+
+final class UpdatedOrderProvider
+{
+    public function __construct(private FormFactoryInterface $formFactory)
```

Comment:
```suggestion
    public function __construct(private readonly FormFactoryInterface $formFactory)
```

Just a small 'best practice' I always do